tests/unit/s2n_server_new_session_ticket_test.c

```diff
@@ -305,6 +338,49 @@ int main(int argc, char **argv)
         }
     }
 
+    /* s2n_tls13_server_nst_read */
```

Comment:
You probably also want a self-talk test. Can we receive the NST message we send? You can't verify the contents of the ticket itself, but you could verify the plaintext values like nonce and lifetime, and at least sanity check the parsing.

Reply:
So technically, we can't actually read the nst messages we write. So I could write a self talk test, but I'd have to move the read_cursor forward a bit in the beginning.

Reply:
:upside_down_face:

That works.